spec/mysql2/statement_spec.rb

```diff
@@ -3,6 +3,7 @@
 
 RSpec.describe Mysql2::Statement do
   before :each do
+    @client.close
```

Comment:
[spec/spec_helper.rb](https://github.com/brianmario/mysql2/blob/master/spec/spec_helper.rb#L27) already created a client.  This one creates another one with a change in its configuration.